### ISSUE TITLE
Cannot resolve URLs

### ISSUE THREAD

**Issue by tdiekel** [Kovah/LinkAce#382]
**Describe the bug**
LinkAce/cURL is not able to resolve URLs. I'm running LinkAce Simple via Portainer. I wrote the .env file previously to the filesystem. All checks are green. The problem is only with resolving URLs since pinging 8.8.8.8 works from inside the container but pinging google.de not. 

I assume the problem is that I've got a pi-hole installed on my system. Thats also why I needed to change the port (to 85). 

**LinkAce setup (please complete the following information):**
 - Version: 1.9.1
 - Installed via: Portainer (Docker) 
 - OS: Debian
 - System: Raspberry Pi 4 

**Additional context**
[laravel-2022-02-02.log](https://github.com/Kovah/LinkAce/files/7990451/laravel-2022-02-02.log)

**Comment by tdiekel** [comment]
Okay I've found the issue. I set the pi-hole to "Allow only local requests" which allows only queries from devices that are at most one hop away (local devices). Changing this to "Permit all origins" 

Is there a way to avoid this? 

Some debug outputs:

```sh
/app # ip a
1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue state UNKNOWN qlen 1000
    link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00
    inet 127.0.0.1/8 scope host lo
       valid_lft forever preferred_lft forever
283: eth0@if284: <BROADCAST,MULTICAST,UP,LOWER_UP,M-DOWN> mtu 1500 qdisc noqueue state UP
    link/ether 02:42:c0:a8:70:03 brd ff:ff:ff:ff:ff:ff
    inet 192.168.112.3/20 brd 192.168.127.255 scope global eth0
       valid_lft forever preferred_lft forever
```

```sh
/app # nslookup google.de
Server:         127.0.0.11
Address:        127.0.0.11:53

Non-authoritative answer:

Name:   google.de

Address: 216.58.212.163

Non-authoritative answer:
Name:   google.de
Address: 2a00:1450:4001:800::2003
```



tl;dr Change pi-hole settings 
Settings -> DNS -> Interface settings -> Permit all origins-

**Comment by Kovah** [comment]
I guess the only way to solve this with your Pi Hole setup would be to use the host network for the LinkAce Docker container. This way the container would get his own Ip from your router/switch and act as a standalone device. But you would not be able to use a proxy in front of it.

**Comment by martadinata666** [comment]
I assume the pi hole also in docker? As it need permit all origin. If that is the case then nothing you can do, as it how docker network work. host -> docker bridge -> pihole, so the pihole != host ip.

**Comment by tdiekel** [comment]
> I guess the only way to solve this with your Pi Hole setup would be to use the host network for the LinkAce Docker container. This way the container would get his own Ip from your router/switch and act as a standalone device. But you would not be able to use a proxy in front of it.

I did try this with `network_mode: host` in the docker compose but got some error. Will try it later again.

> I assume the pi hole also in docker? As it need permit all origin. If that is the case then nothing you can do, as it how docker network work. host -> docker bridge -> pihole, so the pihole != host ip.

No the pihole is installed directly on the device. So pihole = host ip.

**Comment by tdiekel** [comment]
> > I guess the only way to solve this with your Pi Hole setup would be to use the host network for the LinkAce Docker container. This way the container would get his own Ip from your router/switch and act as a standalone device. But you would not be able to use a proxy in front of it.

> I did try this with `network_mode: host` in the docker compose but got some error. Will try it later again.

The setting `network_mode: host` is not working since the pihole runs on port 80 and host mode ignores port settings. 
https://docs.docker.com/network/host/

Setting `network_mode: bridge` solves the problem. But I don't know why because `bridge` should be the default.
